src/pages/markets/basket-indices/index.tsx

```diff
@@ -20,11 +21,12 @@ const BasketPage = () => (
         <DerivedFXHero
             title={localize('Derived')}
             description={localize(
-                'Trade on asset prices derived from real-world or simulated markets. Manage your exposure by selecting the volatility level to suit your risk appetite. Choose from our 24/7 synthetics, derived FX indices, and basket indices.',
+                'Trade on asset prices derived from real-world or simulated markets. Manage your exposure by selecting the volatility level to suit your risk appetite. Choose from our 24/7 synthetics, derived FX indices, and basket indices.'
```

Comment:
it seems your prettier / eslint is not working, on save there should be a trailing slash at the end of the line. 
can you please check it out? 
if we merge this, the next person that saves this file will apply the trailing slash change.

Reply:
i check it, thanks